### ISSUE TITLE
Altdorf doors prevent aoe going through

### ISSUE THREAD

**Issue by Vayra** [WarEmu/WarBugs#15750]
**Expected behavior**
Being able to shoot non-target and ground-targeted aoes in and through doors/gates in Altdorf

**Actual behavior:**
Doors in stage 3 and gate on stairs near catapults in stage 1/2 do not allow you to shoot non-targeted aoe (like Infernal Wave or Ice Spikes on sorc) or ground-targeted aoe (Pit of Shades) through them. GTAOE just fail to start even if the targeting marker shows a valid location, and non-targeted aoes simply don't affect anyone in or behind the door/gate. 

**Steps to reproduce the problem:**
Stand on one side of the doors in stage 3 and try to use Pit of Shades/BW equivalent through to the other side and inside the doorway, then try to hit an opposing player with a non-targeted aoe attack.

**Comment by RoR-Zomega** [comment]
To clarify (@Vayra correct me if you mean different doors and I'll do a bug report for this one) but the area specifically is:

Altdorf's palace on stage 3 (i.e. destro won stage 2) and it's the 2 doors that separate the inside (destro area with throne room) from the outside (order area in the courtyard). Point blank AoEs won't hit enemies on the other side of the door even if there's no obvious LOS issues.

**Comment by Vayra** [comment]
Yes, that is one area. The other is the open gate in the middle of the stairs leading up to the catapults in stage 1.  Has the same issue.

**Comment by GabrielB08** [comment]
I had the same issue with Magus Aoe, both ground target and Pandemonium; which built for 1 sec and then either froze or just didn't cast. Not sure if this helps, but I hope it does!

**Comment by andytomlins** [comment]
Lifetaps also do not work through doors in Altdorf or IC. The casts begin but are cancelled by the los check even when doors are open.
